### PR TITLE
[Backoffice] Implement Content Security Policy headers.

### PR DESCRIPTION
**Pull request by NoUseFreak**

### CODE REVIEW

File: src/Clastic/BackofficeBundle/EventListener/ContentSecurityPolicyListener.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    private function buildOptions()
+    {
+        $options = [
+            'default-src' => '',
+            'img-src'     => 'https://secure.gravatar.com',
+            'font-src'    => 'https://netdna.bootstrapcdn.com',
+
+            // Needed for CKeditor
+            'style-src'   => '\'unsafe-inline\'',
```
Comment (NoUseFreak) [inline src/Clastic/BackofficeBundle/EventListener/ContentSecurityPolicyListener.php]: Settings are currently very open due to ckeditor.

Comment (NoUseFreak) [inline src/Clastic/BackofficeBundle/EventListener/ContentSecurityPolicyListener.php]: @dvaneven Do you know a solution for this?

**Comment by dvaneven** [comment]
Fixed?

**Comment by NoUseFreak** [comment]
Merged. Better have some loose security than none.

**Comment by dvaneven** [comment]
true

**Pull request closed without merging**